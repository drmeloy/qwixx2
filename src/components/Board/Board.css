.Board {
  border: 1px solid;
}