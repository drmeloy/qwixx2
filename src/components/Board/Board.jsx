import React from 'react';
import DiceDisplay from '../DiceDisplay/DiceDisplay';
import styles from './Board.css';

export default function Board() {
  return (
    <>
      <div className={styles.Board}>
        <DiceDisplay />
      </div>
    </>
  );
}
