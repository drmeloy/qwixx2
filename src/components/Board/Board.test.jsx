import React from 'react';
import { shallow } from 'enzyme';
import Board from './Board';
import { Provider } from 'react-redux';
import store from '../../store';

describe('Board component', () => {
  it('Matches snapshot', () => {
    const wrapper = shallow(<Provider store={store}><Board /></Provider>);
    expect(wrapper).toMatchSnapshot();
  });
});
