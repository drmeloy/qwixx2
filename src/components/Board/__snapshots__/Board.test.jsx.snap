// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board component Matches snapshot 1`] = `
<h1>
  Board board board!
</h1>
`;
